src/Plutarch/Test/QuickCheck/Helpers.hs

```diff
@@ -5,6 +5,10 @@ module Plutarch.Test.QuickCheck.Helpers (loudEval) where
 import Plutarch (ClosedTerm, Config (..), TracingMode (DoTracing))
 import Plutarch.Evaluate (evalTerm)
 
+{- | Evaluates the 'ClosedTerm' and put evaluation result back into the 'ClosedTerm'.
+
+ @since 2.1.6
+-}
```

Comment:
"into a 'ClosedTerm'"

Also, it might be worth leaving a note explaining why this exists: specifically, you can 'burn in' arguments for something like a property test.

Reply:
Yeah, good idea